spyder_profiler/__init__.py

```diff
@@ -3,4 +3,8 @@
 #==============================================================================
 # The following statement is required to register this 3rd party plugin:
 #==============================================================================
+from spyder.utils.qthelpers import qapplication
+# The line below is needed in order to prevent a segmentation fault.
+# See issue #4014
+app = qapplication()
```

Comment:
Yeah that's what I was trying to do :). That's why I'm importing `qapplication`, or maybe I misunderstood the docstring?
```
    """
    Return QApplication instance
    Creates it if it doesn't already exist
    
    test_time: Time to maintain open the application when testing. It's given
    in seconds
    """
```